packages/combo-box/test/fixtures/mock-combo-box-light-template-wrapper.js

```diff
@@ -1,4 +1,5 @@
 import '../../vaadin-combo-box-light.js';
+import '@vaadin/text-field';
```

Comment:
I had to import `<vaadin-text-field>` here since it's being used in the combo box light below. It was also the cause of failure in the tests in the `item-template` suite because the target element didn't have any size and the overlay was being closed. 

Can you review this change, @web-padawan?

Reply:
LGTM 👍